packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/BitcoinOptions.tsx

```diff
@@ -9,6 +9,8 @@ import { OpenGuideFromTooltip } from 'src/components/guide';
 import { Locktime } from './Locktime';
 import { CoinControl } from './CoinControl/CoinControl';
 import { OnOffSwitcher } from '../OnOffSwitcher';
+import { selectNetworkBlockchainInfo } from '@suite-common/wallet-core';
+import { useSelector } from '../../../../../../hooks/suite';
```

Comment:
```suggestion
import { useSelector } from 'src/hooks/suite';
```

Reply:
I prefer relative paths as they are IMO better supported by IDEs and tooling and don't require any special configuration. 

Do we have any team consensus on this?

Reply:
There are some unspoken rules regarding import ordering (should be automated though) and relative vs. absolute imports which prefer absolute imports unless the file is in the same folder or subfolder. We can discuss at code improvements. I have no hard preference, but would like to be consistent.

Reply:
okok, i have the other peference, but its not really strong. but it would be nice to have it somehow automated and autofixed. thinking about import formatting is annoying